flex-checks/src/main/resources/org/sonar/l10n/flex/rules/flex/ClassComplexity.html

```diff
@@ -1,4 +1,5 @@
-<p>
-The cyclomatic complexity of a class should not exceed a defined threshold.
-Complex code can perform poorly and will in any case be difficult to understand and therefore to maintain.
-</p>
+<p>The cyclomatic complexity of a class should not exceed a defined threshold. Complex code can perform poorly and will in any case be difficult to
+understand and therefore to maintain.</p>
+<h2>Deprecated</h2>
+<p>This rule is deprecated, and will eventually be removed.</p>
```

Comment:
Deprecated? Are we going to implement cognitive complexity?

Reply:
My understanding is that this deprecation is not related to cognitive complexity. This is confirmed by the [SONARFLEX-86](https://jira.sonarsource.com/browse/SONARFLEX-86) which was created by Ann.